htmd/parameterization/util.py

```diff
@@ -172,28 +172,36 @@ def getFixedChargeAtomIndices(mol, fix_charge):
     return fixed_atom_indices
 
 
-def minimize(mol, qm, outdir):
+def minimize(mol, qm, outdir, min_type='qm', mm_minimizer=None):
```

Comment:
This looks ugly! We really need to refactor calculator and minimizer classes.

Reply:
agreed